### DISCUSSION TITLE
Localizing aria-label for clear text button in Searchbox

### DISCUSSION THREAD

**Discussion by tripsus** [microsoft/fluentui#24379] [Q&A] [answered]
We have an accessibility where aria-label for cross button(X) in Search box is not localized. While using narrator for Italian language we see that text corresponding to cross button is announced in English.
While I am aware that we can override aria-label for clearButtonProps in Search box but wouldn't it be easier if aria-label for clearButton where localized by SearchBox implementation itself.

**Comment by micahgodbolt** [comment, marked as answer]
We typically provide a default string in english and leave localization up to the needs of the individual application. Adding localization into each component would increase bundle size and complexity.
